src/benchq/resource_estimation/openfermion_re.py

```diff
@@ -1,69 +1,92 @@
 ################################################################################
 # © Copyright 2022 Zapata Computing Inc.
 ################################################################################
-from typing import Any, Dict
+from typing import Any, Dict, Tuple
 
 import numpy as np
-import pyscf
 from openfermion.resource_estimates import sf
 from openfermion.resource_estimates.surface_code_compilation.physical_costing import (
     cost_estimator,
 )
 
+from benchq.resource_estimation._compute_lambda_sf import compute_lambda
 
-def model_toffoli_and_qubit_cost_from_single_factorized_mean_field_object(
-    mean_field_object, rank, DE, CHI
-):
-    num_orb = len(mean_field_object.mo_coeff)
+
+def get_single_factorized_qpe_toffoli_and_qubit_cost(
+    h1: np.ndarray,
+    eri: np.ndarray,
+    rank: int,
+    allowable_phase_estimation_error: float,
```

Comment:
Are the reasonable defaults for some of the parameters in this function? For example, `allowable_phase_estimation_error`?